app/views/docs/functions.phtml

```diff
@@ -384,6 +384,35 @@ func main(req: RequestValue, res: RequestResponse) throws -> RequestResponse {
     <p>If you're running Appwrite locally and trying to connect to your local Appwrite instance from a cloud function, you'll have to use your local IP address in place of localhost. This is because, within the container localhost does not refer to the host machine rather the container itself. </p>
 </div>
 
+<h2><a href="/docs/functions#ignoreFiles" id="ignoreFiles">Ignore Files</a></h2>
+
+<p>Library folders such as <b>node_modules</b> or <b>vendor</b> should not be included in the tarball, although you will want them locally so you have syntax highlighting in your IDE. Similarly, you should <b>not</b> include files containing secrets in your deployment. You can use the Appwite CLI's <b>file ignoring</b> feature to exclude specific files from a deployment.</p>
+
+<p>You can open the <b>appwrite.json</b> file and look for your function's definition. In there, you can set the value for the <b>ignore</b> key to specify which files and folders should be ignored. This value must be an array, as seen in the example below:</p>
+
+<div class="ide margin-bottom" data-lang="json" data-lang-label="JSON">
+    <pre class="line-numbers"><code class="prism language-json" data-prism>{
+    ...
+    "functions": [
+        {
+            "$id": "6213b58cb21dda6c3263",
+            "name": "My Awesome Function",
+            "runtime": "node-17.0",
+            "path": "My Awesome Function",
+            "entrypoint": "src/index.js",
+            "ignore": [ "node_modules", ".env" ]
+        },
+        ...
+    ],
+}</code></pre>
+</div>
+
+<p>The example configuration above would not deploy the folder <b>node_modules</b> and the file <b>.env</b>.</p>
+
+<p>Alternatively, you can add a <b>.gitignore</b> file into your function folder and Appwrite CLI will ignore files specified in there. Keep in mind that the <b>ignore</b> configuration in <b>appwrite.json</b> must not be present for <b>.gitignore</b> to be used.</p>
```

Comment:
```suggestion
<p>Alternatively, you can add a <b>.gitignore</b> file into your function folder and Appwrite CLI will ignore files specified in there. Keep in mind that if present, the <b>ignore</b> configuration in <b>appwrite.json</b> overrides the <b>.gitignore</b> file.</p>
```

Reply:
Used yours, but changed a little. This could indicate we write to `.gitignore` which we don't.

`Keep in mind that if present, the <b>ignore</b> configuration in <b>appwrite.json</b> nullify the <b>.gitignore</b> file configuration.`

Reply:
```
Keep in mind that if present, the <b>ignore</b> configuration in <b>appwrite.json</b>will nullify the <b>.gitignore</b> file configuration.
```